src/autopas/containers/cellPairTraversals/C18BasedTraversal.h

```diff
@@ -41,19 +41,24 @@ class C18BasedTraversal : public CBasedTraversal<ParticleCell, PairwiseFunctor,
   /**
    * The main traversal of the C18Traversal.
    * @copydetails C01BasedTraversal::c01Traversal()
+   * @tparam allCells Defines whether or not to iterate over all cells with the loop body given as argument. By default
+   * it will not iterate over all cells and instead skip the last few cells, because they will be covered by the base
```

Comment:
From this description it is not directly clear to me:
1) how much is "last few cells" (suppose last row * overlap?)
2) when would I set this to true / false
maybe then also rename it accordingly?

Reply:
more explanations added, but no rename done, as the name is descriptive